src/cmd/ksh93/meson.build

```diff
@@ -24,7 +24,7 @@ libksh = library('ksh', ksh93_files,
                  include_directories: [configuration_incdir, ksh93_incdir],
                  c_args: shared_c_args,
                  dependencies: [libm_dep, libexecinfo_dep, libdl_dep, libsocket_dep, libnsl_dep],
-                 install: false)
+                 install: true)
```

Comment:
It should be built and linked statically. Are you building it as shared library ?

Reply:
Sorry, I missed the commit message. This line should be written as:

```
install: get_option('default_library') == 'shared')
```

Reply:
Yea, you're right. Fixed.